docs/source/projects.rst

```diff
@@ -375,18 +375,59 @@ in the Databricks docs
 of how to use the feature is as follows:
 
 1. Create a JSON file containing the
-`new cluster specification <https://docs.databricks.com/api/latest/jobs.html#jobsclusterspecnewcluster>`_
-for your run. For example:
+`new cluster specification <https://docs.databricks.com/dev-tools/api/latest/jobs.html#jobsclusterspecnewcluster>`_
+or `cluster specification <https://docs.databricks.com/dev-tools/api/latest/jobs.html#clusterspec>`_
+for your run. If you do not need to specify libraries installed on the Spark worker nodes, you can use the
```

Comment:
```suggestion
for your run. Note that running projects against an existing cluster is not supported. If you do not need to specify libraries installed on the Spark worker nodes, you can use the
```

I think it'd be helpful to clarify this in the docs, as [ClusterSpec](https://docs.databricks.com/dev-tools/api/latest/jobs.html#clusterspec) allows you to specify an existing cluster ID, although users will also see an error message if they attempt to run against an existing cluster (that error message I suspect will say something cryptic about "shell command tasks" instead of "mlflow projects", will file a separate PR to fix that...)